docs/stylesheets/extra.css

```diff
@@ -68,3 +68,9 @@
 .md-typeset .admonition {
   font-size: 100%;
 }
+
+/* Giving h4 tag distinctive style because it is non-differntial with bold text */
+.md-typeset h4 {
+  font-weight: 400;
```

Comment:
`font-weight: 400` is the same as normal text (see https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#values), so Im not sure this is really an improvement :(.

Reply:
The #3696 issue it was stated that bold text and h4 tags context must have distinct style so that it can be recognized easily
that's why I came up with this

Reply:
yes, but this makes it so that the h4 tags are impossible to distinguish from regular text, which is even worse than being hard to distinguish from bold text?

Reply:
maybe increasing font-size may work initially i thinked about that but thought if styling can make difference